tests/conftest.py

```diff
@@ -499,6 +499,32 @@ def db_context(db_config, db_host, db_port, db_name, db_conn):
     )
 
 
+@pytest.fixture
+def tendermint_host():
+    return os.getenv('TENDERMINT_HOST', 'localhost')
+
+
+@pytest.fixture
+def tendermint_port():
+    return int(os.getenv('TENDERMINT_PORT', 46657))
+
+
+@pytest.fixture
+def tendermint_ws_url(tendermint_host, tendermint_port):
+    return f'ws://{tendermint_host}:{tendermint_port}/websocket'
```

Comment:
Nice! I didn't know about `f` strings.

Reply:
Thanks to @kansi for using them first!

Reply:
We need to be careful about using `f` strings for now. ReadTheDocs is failing to build the docs in the `tendermint` branch and I think it's because it uses an older version of Python 3.x that doesn't understand `f` strings. Not sure. But I took a look at the ReadTheDocs error logs for building the `tendermint` branch, and it was saying syntax error on a line with an `f` string.

Reply:
We can use the `build` option of readthedocs (http://docs.readthedocs.io/en/latest/yaml-config.html#build-image) to use Python 3.6.